docs/USAGE.md

```diff
@@ -313,6 +314,11 @@ In case of an error the plugin returns an error code.
 | 11         | The app couldn't establish a connection                                                                                                                                                                    |
 | 1000+      | A connection was established, but an error occurred. The error code is 1000 + the HTTP code, e.g. 1401 means [Unauthenticated](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors). |
 
+### Quick Access Device Controls
+
+[On Android 11 and above](https://www.android.com/intl/en_US/android-11/#a11-device-controls-article) you can long press the power button to control openHAB Items.
```

Comment:
Referring to the power button may be a little too specific ... In Android 12 they moved device controls to the quick settings tiles :-/